src/app/src/mainwindow.cpp

```diff
@@ -270,7 +272,7 @@ void MainWindow::on_durationEdit_valueChanged(double value) {
     m_duration = value;
     m_expSettings.expTimeMS = (1 / m_fps) * 1000;
     m_expSettings.frameCount = m_duration * m_fps;
-    m_camera->SetupExp(m_expSettings);
+    m_camera->UpdateExp(m_expSettings);
```

Comment:
same as `frameRateEdit` probably don't need to update the settings in the camera class until an acquisition is started